src/ASM/ASMs3D.C

```diff
@@ -1622,7 +1622,7 @@ Vec3 ASMs3D::getCoord (size_t inod) const
 }
 
 
-bool ASMs3D::getElementCoordinates (Matrix& X, int iel) const
+bool ASMs3D::getElementCoordinates (Matrix& X, int iel, bool) const
```

Comment:
Wouldn't you need the same `forceItg` stuff here in the 3D?

Reply:
separate PR.